### ISSUE TITLE
Re-organize wiki menu and navigation

### ISSUE THREAD

**Issue by mpalmsten** [Coastal-Imaging-Research-Network/Forum-Wiki#13]

**Comment by mpalmsten** [comment]
I'm working on re-organizing the menu and adding navigation links. Still a work in progress.

**Comment by allisonpenko** [comment]
Reorganized menu and added a Wiki home page. The Wiki home is mirrored to the "Home" page in the Forum-Wiki repository. Changes have to be made to both the "Home" and "CIRN Wiki Home" pages

**Comment by allisonpenko** [comment]
"The Designing a Station" section still needs to be filled out

**Comment by SRHarrison** [comment]
I recently read through and tested all of the demo software in the "Designing a Station" section. 
The content is more organized and reads smoothly. I think it is a nice improvement over the "Optical Systems" / "General Workflow" section which contains similar content. I went ahead and dismantled [Optical Systems](https://github.com/Coastal-Imaging-Research-Network/Forum-Wiki/wiki/Optical-Systems), and relocated useful content (if you don't like the change, I have a backup prior to my edits). 

E.g. Cameras, Lenses, Triggers, Cables and Switches were moved to the 'Hardware Corner'. FOV is superseded by the new description in Designing a Station. General Workflow was removed. 'Cookbook' was tacked onto the end of Designing a Station. 
Fixed Mounting Platforms and Temporary Towers are nicely described in the Designing a Station section, but there is now no mention of UAVs. Where to put them?

A description of pixel tools should be added somewhere, as referenced in [Installation Design](https://github.com/Coastal-Imaging-Research-Network/Forum-Wiki/wiki/Installation-Design): "Pixel tools are described [elsewhere]".
